docs/footers/core_docs.md

```diff
@@ -0,0 +1,336 @@
+## Examples
```

Comment:
There's a lot of content in here that maintainers need to work on. it's strange for it to be "footers". That name is only useful when you think of the content in terms of how we generate stardoc, not when you're thinking in the domain being documented.

I think the nicest way to maintain this is
docs/examples.md
docs/stamping.md
docs/embedding.md
etc

Reply:
I see your point. Since there are identically named sections across multiple rst files, I'll create subdirectories corresponding to each rst and separate out the sections in there. I'll rewrite the genrule to concatenate an array of files of variable size.